tests/resources/VCH-Util.robot

```diff
@@ -159,6 +159,22 @@ Check For The Proper Log Files
     Should Contain  ${output}  ${container}/vmware.log
     Should Contain  ${output}  ${container}/tether.debug
 
+Scrape Logs For the Password
```

Comment:
I've added this test at the end of the regression suite, so that'll take care of checking logs after a bunch of docker commands have been run. Should we still add a docker command inside `Scrape Logs For the Password`?